core/domain/user_services_test.py

```diff
@@ -1732,27 +1751,24 @@ def test_create_user_contributions(self):
         contributions = user_services.get_user_contributions(user_id)
 
         self.assertEqual(
-            ['exp1', 'exp2', 'exp3'],
+            ['expectedId1', 'expectedId2', 'expectedId3'],
             contributions.created_exploration_ids)
 
         self.assertEqual(
-            ['exp2', 'exp3', 'exp4'],
+            ['expectedId2', 'expectedId3', 'expectedId4'],
             contributions.edited_exploration_ids)
 
     def test_update_user_contributions(self):
-        auth_id = 'someUser'
-        user_email = 'user@example.com'
-        created_exp_ids = ['exp1', 'exp2', 'exp3']
-        edited_exp_ids = ['exp2', 'exp3', 'exp4']
-
-        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        created_exp_ids = ['expectedId1', 'expectedId2', 'expectedId3']
```

Comment:
Here and elsewhere ... I'm not entirely sure why you're changing all these to have "expected" in the string. It's a bit misleading because these are the "actual" values for the tests.

When I said:

> Inline the expected value. If you want to make it its own variable, then "expected" should be somewhere in the name.

over [here](https://github.com/oppia/oppia/pull/15356/files/f163020664dcff526280a3ccff648804bbf20a0b#diff-5e15dd19d220cb0c4f0b440590daa03751cf6fecf884f15aed0dfc94e504b3bf), I meant making that test something like this:

```
    def test_add_created_exploration_id(self):
        auth_id = 'someUser'
        user_email = 'user@example.com'

        user_id = user_services.create_new_user(auth_id, user_email).user_id

        user_services.add_created_exploration_id(user_id, 'exp1')

        contributions = user_services.get_user_contributions(user_id)

        expected_created_exp_ids = ['exp1']
        self.assertEqual(
            expected_created_exp_ids,
            contributions.created_exploration_ids)
```

so that only the "expected" part of the assertEqual has the "expected" connotation. Or, alternatively:

```
        self.assertEqual(
            ['exp1'],
            contributions.created_exploration_ids)
```

will do.

Does this make sense? The original names were fine, I was talking mainly about the variable name that you're using -- notice in the first example above I only use this before the assertion and not as part of the setup logic.

Reply:
Ah I see, I apologize I was misunderstanding the request. I inlined all the expected values in assertions here and in other related tests!